app/uk/gov/hmrc/cataloguefrontend/connector/ServiceDependenciesConnector.scala

```diff
@@ -127,24 +127,25 @@ class ServiceDependenciesConnector @Inject() (
       .get(url"$servicesDependenciesBaseUrl/api/repository-name?group=$group&artefact=$artefact&version=${version.toString}")
       .execute[Option[String]]
 
-  def getRepositoryModules(repositoryName: String)(implicit hc: HeaderCarrier): Future[Option[RepositoryModules]] = {
+  def getRepositoryModulesLatestVersion(repositoryName: String)(implicit hc: HeaderCarrier): Future[Option[RepositoryModules]] = {
     implicit val dr = RepositoryModules.reads
     httpClientV2
-      .get(url"$servicesDependenciesBaseUrl/api/module-dependencies/$repositoryName")
-      .execute[Option[RepositoryModules]]
+      .get(url"$servicesDependenciesBaseUrl/api/repositories/$repositoryName/module-dependencies?version=latest")
+      .execute[Seq[RepositoryModules]]
+      .map(_.headOption)
   }
 
-  def getRepositoryModules(repositoryName: String, version: Version)(implicit hc: HeaderCarrier): Future[Option[RepositoryModules]] = {
+  def getRepositoryModules(repositoryName: String, version: Version)(implicit hc: HeaderCarrier): Future[Seq[RepositoryModules]] = {
```

Comment:
this should also return `Option` - it's a duplication of `getRepositoryModulesLatestVersion` - can we combine for an `Option[Version]` which will default to "latest" if not present?